src/main/java/frc/robot/Robot.java

```diff
@@ -142,6 +142,58 @@ public void robotInit() {
                 }, Set.of(armTelescopicSystem, clawJointSystem, clawGripperSystem)
         );
 
+
+        Command collectFromSourceCommand = new ParallelCommandGroup(
```

Comment:
you didn't attach these to buttons and you didn't add `ClawGripperOuttake` and `ClawGripperIntake` to buttons.